app/session.go

```diff
@@ -93,18 +93,26 @@ func (a *App) GetSessions(userId string) ([]*model.Session, *model.AppError) {
 	return a.Srv.Store.Session().GetSessions(userId)
 }
 
+func (a *App) revokeSession(session *model.Session) *model.AppError {
+	if session.IsOAuth {
+		a.RevokeAccessToken(session.Token)
+	} else {
+		if err := a.Srv.Store.Session().Remove(session.Id); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
 func (a *App) RevokeAllSessions(userId string) *model.AppError {
 	sessions, err := a.Srv.Store.Session().GetSessions(userId)
 	if err != nil {
 		return err
 	}
 	for _, session := range sessions {
-		if session.IsOAuth {
-			a.RevokeAccessToken(session.Token)
-		} else {
-			if err := a.Srv.Store.Session().Remove(session.Id); err != nil {
-				return err
-			}
+		revokeErr := a.revokeSession(session)
```

Comment:
I'm not sure there's any value in breaking this out to a separate function if it's now only called in one place (but 0/5 on this).

Reply:
definitely not needed anymore, it made sense with the original set of changes. I did remove most of the original code but I missed this one. Good catch!